Sources/UI/KPMediaPlayer.swift

```diff
@@ -378,6 +379,32 @@ extension KPMediaPlayer {
         }
     }
     
+    private func handleAdEvents() {
```

Comment:
Why was this separated? The adStarted and adComplete events are already in the registerAdEvents function.
And please use the type alias KPAdEvent and not the Playkit AdEvent.